### DISCUSSION TITLE
How would you block a Websub hub sending too large body?

### DISCUSSION THREAD

**Discussion by marienfressinaud** [FreshRSS/FreshRSS#6996] [Q&A]
Hey there :wave: 

For a few weeks now, I have had a recurring error in my log:

```
[ nginx ]  2024/11/13 00:35:15 [error] 785#785: *17254 client intended to send too large body: 1411759 bytes, client: <hub ip>, server: rss.flus.fr, request: "POST /api/pshb.php?k=<key> HTTP/1.1", host: "rss.flus.fr"
```

I could increase the accepted size for the body, but that would be a short term fix. I would instead tell FreshRSS not to subscribe to this particular feed with Websub. I can't remember if FreshRSS already allows this, but I don't think so. How would you handle this case? or should we develop a fix?

I cannot share the feed for privacy reasons as it belongs to one of my users, but it is a Blogger feed, subscribed via a pubsubhubbub.appspot.com hub.

**Comment by Frenzie** [comment]
What concretely is responsible for making it so large?

**Reply by marienfressinaud** [reply]
The feed contains some images encoded in base64 :grimacing:

**Comment by Alkarex** [comment]
There is no option to selectively opt out for WebSub for specific feeds or domains at the moment, but that should be fairly easy to add
